### ISSUE TITLE
train sam loss isn't focal loss.

### ISSUE THREAD

**Issue by shipengai** [UX-Decoder/Semantic-SAM#24]
As the code show, the training loss is ce loss not focal loss
`losses = {
            "loss_mask_bce_0": sigmoid_ce_loss_jit(point_logits, point_labels, num_masks),
            "loss_mask_dice_0": dice_loss_jit(point_logits, point_labels, num_masks),
        }`

**Comment by FengLi-ust** [comment]
This is a typo. Thanks for your feedback, and we will update the paper later.
